python/ray/ml/utils/test_utils.py

```diff
@@ -0,0 +1,90 @@
+import os
+import re
+import shutil
+import tempfile
+
+from ray.ml.storage import Storage
+
+
+class LocalTestStorage(Storage):
```

Comment:
Will this be used as a test mock for remote storage case? 
Currently `is_cloud_target=False` gates this from being used as a mock.